apbs/test/test_chemistry/test_atom_list.py

```diff
@@ -1,76 +1,73 @@
 from apbs.geometry import Coordinate
-from apbs.chemistry import AtomList, CellList
-import pytest
+from apbs.chemistry import Atom, AtomList
+from apbs.pqr import PQRReader
+from pytest import approx, fixture
 
 
-@pytest.fixture()
-def pdb_file(tmp_path):
+@fixture
+def get_atom_list(tmp_path) -> AtomList:
+    """Return the AtomList from the file generated."""
 
-    fp = tmp_path / "atom.pdb"
+    fqpn = tmp_path / "atom.pdb"
 
-    with open(fp, "w") as f:
+    with open(fqpn, "w") as f:
         f.write(
-            "HETATM    1  C    ALK    1       "
+            "HETOM    1  C    ALK    1       "
```

Comment:
"HETOM" is not a PDB keyword.  "HETATM" is correct.

Where did this change come from?  What are you testing against?  This is the best reference:  https://www.wwpdb.org/documentation/file-format-content/format33/v3.3.html

Reply:
Fixed